pass-data-client/src/main/java/org/eclipse/pass/support/client/JsonApiPassClient.java

```diff
@@ -92,17 +97,41 @@ public JsonApiPassClient(String baseUrl, String user, String pass) {
         }
 
         client = client_builder.build();
+        moshi = create_moshi(false);
+
+        // Serialize null value of attributes for the JSON API document
+        update_moshi = create_moshi(true);
+    }
 
+    private Moshi create_moshi(boolean serialize_nulls) {
```

Comment:
Instead of passing a boolean flag to change the behavior of the method, what do you think about implementing a new method name `create_moshi_serialize_nulls`?  This method can get the initial factory/builder, create the serialize_nulls_factory and add it to the build, the call `create_moshi` to finish adding the rest?